litmus-portal/frontend/src/views/Settings/TeamingTab/Invitation/AcceptedInvitations/index.tsx

```diff
@@ -48,45 +56,24 @@ const AcceptedInvitations: React.FC = () => {
   });
 
   useEffect(() => {
-    const otherProject: Project[] = [];
+    const otherProject: OtherProjectsType[] = [];
     projects.map((project) => {
       return project.members.forEach((member: Member) => {
         if (
           member.user_id === userID &&
           member.role !== 'Owner' &&
           member.invitation === 'Accepted'
```

Comment:
string can be replaced with constant

Reply:
I have already made the changes, in an upcoming PR